pynopegl/init_header.py

```diff
@@ -195,11 +195,15 @@ def _args_setter(self, cython_setter, param_name, *args):
         return cython_setter(param_name, args)
 
     def _add_nodes(self, param_name, *nodes):
+        if not nodes:
+            return 0
         if hasattr(nodes[0], "__iter__"):
             raise Exception(f"add_{param_name}() takes elements as positional arguments, not list")
         return self._param_add_nodes(param_name, len(nodes), nodes)
 
     def _add_f64s(self, param_name, *f64s):
+        if not nodes:
```

Comment:
I believe it should be `if not f64s`

Also, can you update the commit message to something more generic like: `pynopegl: allow empty argument for list-based parameters`?

Reply:
D'oh!